mbuild/conversion.py

```diff
@@ -747,7 +748,12 @@ def save(compound,
         Depending on the file extension these will be passed to either
         `write_gsd`, `write_hoomdxml`, `write_lammpsdata`,
         `write_mcf`, or `parmed.Structure.save`.
-        See https://parmed.github.io/ParmEd/html/structobj/parmed.structure.Structure.html#parmed.structure.Structure.save
+        For the case when saving out a `.gro` and `.top` file using parmed,
+        if the `combine` is not provided in **kwargs, we will default to
+        `combine='all'` to avoid unnecessary error when parmed tries to
+        change the atom order when saving out.
+        For more information, see
+        https://parmed.github.io/ParmEd/html/structobj/parmed.structure.Structure.html#parmed.structure.Structure.save
```

Comment:
This link doesn't say thing about `combine="all"`

Reply:
I will add a link to the parmed save function